docs/examples/prometheus_exemplars/README.rst

```diff
@@ -0,0 +1,107 @@
+``trace_id`` attributes (`specification
+<https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/compatibility/prometheus_and_openmetrics.md#exemplars>`_).
+You can get the current span using :func:`opentelemetry.trace.get_current_span`, then format
+its span and trace IDs to hexadecimal strings using :func:`opentelemetry.trace.format_span_id`
+and :func:`opentelemetry.trace.format_trace_id`.
+
+.. literalinclude:: server.py
+    :language: python
+    :dedent:
+    :start-after: [START opentelemetry_prom_exemplars_attach]
+    :end-before: [END opentelemetry_prom_exemplars_attach]
+
+Then make an observation using a `Prometheus Histogram
+<https://prometheus.io/docs/concepts/metric_types/#histogram>`_ as shown below. Google Cloud
+Monitoring can only display exemplars attached to Histograms.
+
+.. literalinclude:: server.py
+    :language: python
+    :dedent:
+    :start-after: [START opentelemetry_prom_exemplars_observe]
+    :end-before: [END opentelemetry_prom_exemplars_observe]
+
+Run
+---
+
+Checkout the example code if you don't already have the repository cloned:
+
+.. code-block:: sh
+
+  git clone https://github.com/GoogleCloudPlatform/opentelemetry-operations-python.git
+  cd docs/examples/prometheus_exemplars
+
+First, set the environment variables needed to provide authentication to the Collector when it
+runs in docker.
+
+.. code-block:: sh
+
+    export USERID=$(id -u)
+    export PROJECT_ID=<your-gcp-project>
+    export GOOGLE_APPLICATION_CREDENTIALS="${HOME}/.config/gcloud/application_default_credentials.json"
+
+Build and start the example containers using ``docker-compose``:
+
+.. code-block:: sh
+
+    docker-compose up --build --abort-on-container-exit
+
+This starts three containers:
+
+#. The Flask server written in ``server.py``. It receives requests and simulates some work by
+   waiting for a random amount of time.
+#. The OpenTelemetry Collector which receives traces from the Flask server by OTLP and scrapes
+   Prometheus metrics from the Flask server's ``/metrics`` endpoint.
+#. A load generator that sends constant requests to the Flask server.
+
+Checking Output
+---------------
+
+While running the example, you can go to `Cloud Monitoring Metrics Explorer page
+<https://console.cloud.google.com/monitoring/metrics-explorer>`_ to see the results. Click on
+the "Metric" dropdown, type ``my_prom_hist``, and select the metric from under "Prometheus
+Target". The full metric name is ``prometheus.googleapis.com/my_prom_hist_seconds/histogram``.
+
+.. image:: select_metric.png
+  :alt: Select the metric
+
+After selecting the metric, you should see something like the image below, with a heatmap
+showing the distribution of request durations in the Python server.
+
+.. image:: heatmap.png
+  :alt: Metrics explorer heatmap
+
+The circles on the heatmap are called "exemplars", which link to example span that fell within
```

Comment:
Nit: example spans (plural)

Reply:
Reworded to make it clear each circle is a single span

Reply:
Updated, please resolve if it looks good